mypy-requirements.txt

```diff
@@ -1,6 +1,6 @@
 typing_extensions>=3.7.4
 mypy_extensions>=0.4.3,<0.5.0
-typed_ast>=1.4.0,<1.5.0
+typed_ast>=1.4.0,<1.5.0; python_version<'3.8'
```

Comment:
typed_ast currently can't be compiled on python 3.10. This has already been fixed upstream (https://github.com/python/typed_ast/pull/155), but there hasn't been a release yet. Until there has been we just need to not install it.